package.json

```diff
@@ -1,56 +1,56 @@
 {
-	"name": "dogear",
-	"description": "A hapi plugin for sending expanded metrics to statsd-compliant services",
-	"version": "2.0.3",
-	"engines": {
-		"node": ">=4.0.0"
-	},
-	"main": "src/index.js",
-	"scripts": {
+  "name": "dogear",
```

Comment:
There was no benefit to this file to restructure it and cause all the lines to be changed. Please do not explode the diff for the sake of changing style; limit changes to just what needs to be changed to enable the new functionality. Please revert these changes and leave only the relevant package and node support changes.

Reply:
So you really want mixed tabs and spaces?